src/merge-to-release.ts

```diff
@@ -77,6 +98,32 @@ export function getBranchNameReleaseSerialNumber(
   return isNaN(releaseNumber) ? undefined : releaseNumber
 }
 
+/**
+ * Determine the serial number of a release branch.
+ * Supports both "release/RLS-001" and "release/1.0.1" formats.
+ *
+ * @export
+ * @param {string} branchName
+ * @param {string} releasePrefix
+ * @param {string} releaseTaskPrefix
+ * @returns {(number | undefined)}
+ */
+export function getBranchSerialNumber(
```

Comment:
good
I see that in the repo we have tests, there are tests for this file, please add your case there